src/anbox/cmds/session_manager.cpp

```diff
@@ -126,7 +126,13 @@ anbox::cmds::SessionManager::SessionManager()
       return EXIT_FAILURE;
     }
 
-    if (!fs::exists("/dev/binder") || !fs::exists("/dev/ashmem")) {
+    auto binder = "/dev/binder";
+    // If available, use own binder to not cause problem with existing binder
+    if (fs::exists("/dev/anbox-binder")) {
```

Comment:
I think we should make the name of the binder node more a configuration option so you can pick for Ubports `-DBINDER_PATH=/dev/anbox-binder` otherwise we default to `/dev/binder`. That avoids having a list of binder nodes in the code which would just get larger if other projects pick a different name.